notebooks/jupyter_dev_example.ipynb

```diff
@@ -1,5 +1,16 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1867d47c-b124-4ae2-93c1-390d9c8578c7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext autoreload\n",
+    "%autoreload 2"
+   ]
+  },
```

Comment:
what does this do? do I care?

Reply:
It means that if you change the sycamore library it will automatically be reloaded in the jupyter notebook. Doesn't hurt anything if you don't. Was critical for making the show fix.